swan-lake/integration-tools/health-tool.md

```diff
@@ -155,3 +208,204 @@ Follow the steps below to use the generated package by running the cloned Baller
+| `--excluded-profile`     | If only a specific profile(s) need(s) to be skipped when generating Ballerina API templates, specify the profile URL as the value of this parameter. This argument can be used more than once.                                                                                                                                                                                                                                                                                                                                     | Optional           |
+
+### Template generation example
+
+Follow the steps below to try out an example template generation use case of the Health tool.
+
+#### Clone the example project
+
+Clone the [artifacts of the example](https://github.com/ballerina-guides/healthcare-samples/tree/main/working-with-health-tool/template-generation) and extract them to a preferred location.
+
+The cloned directory includes the [`ig-uscore/definitions`](https://github.com/ballerina-guides/healthcare-samples/tree/main/working-with-health-tool/template-generation/ig-uscore/definitions) directory, which includes the definition files of the FHIR specification.
+
+#### Generate the template
+
+Follow the steps below to run the Health tool and generate the Ballerina API templates for the selected package.
+
+>**Note:** You need to have a Ballerina package generated using FHIR definitions and the Health tool as a prerequisite for generating the Ballerina API templates for them. This example uses the [`health.fhir.r4.uscore501`](https://central.ballerina.io/ballerinax/health.fhir.r4.uscore501/latest) package in Ballerina Central.
+
+1. Navigate to the cloned `working-with-health-tool/template-generation` directory.
+
+2. Run the tool with the [required arguments](#template-generation-command-options) to generate the Ballerina APIs.
+
+    >**Note:** This example uses the [definitions files](https://github.com/ballerina-guides/healthcare-samples/tree/main/working-with-health-tool/template-generation/ig-uscore/definitions) downloaded from the **JSON Definitions ZIP archive** of the [US Core implementation guide](https://build.fhir.org/ig/HL7/US-Core/downloads.html) to generate the templates.
+
+    ```
+    $ bal health fhir -m template -o ig-uscore/gen --org-name healthcare_samples --dependent-package ballerinax/health.fhir.r4.uscore501 ig-uscore/definitions
+    [INFO] Generating templates for all FHIR profiles...
+    [INFO] Ballerina FHIR API templates generation completed successfully. Generated templates can be found at: /Users/praneesha/Desktop/healthcare-samples/working-with-health-tool/template-generation/ig-uscore/gen
```

Comment:
Can we have the output lines with a bit more simpler/genric path instead of having person names?

Reply:
Can we have the output location as: /Users/healthcare-samples/working-with-health-tool/template-generation/ig-uscore/gen? @nirmal070125 @isuruh15

Reply:
Updated to `/tmp` as per https://github.com/ballerina-platform/ballerina-dev-website/pull/8393#discussion_r1421201400.